### PR TITLE
Add Overview of a CI/CD workflow with Zuul article

### PR DESCRIPTION
**Pull request by TristanCacqueray**

### CODE REVIEW

**Comment by yac** [comment]
good read 👍

File: source/blog/2018-01-22-CICD-workflow-with-zuul.html.md [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+        name: software-factory/sf-config
+    - branch: master
+      change_url: https://softwarefactory-project.io/r/10605
+      project:
+        name: software-factory/sf-ci
+```
+
+In this example, the depends-on list includes three changes:
+* Pages roles added to zuul-jobs-distgit,
+* Pages jobs configured in sf-cofnig, and
```
Comment (morucci) [inline source/blog/2018-01-22-CICD-workflow-with-zuul.html.md]: typo

File: source/blog/2018-01-22-CICD-workflow-with-zuul.html.md [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+  * Job settings can be modified per project, and
+  * *queue* makes all the projects depend on each other automatically.
+
+# Conclusion
+
+Zuul can be used to effectively manage complex continous integration and
+deployment pipelines with powerfull cross repository dependencies management.
+
+This article presented the Software Factory workflow where rpm packages are
+being continously built, tested and delivered through code review.
```
Comment (morucci) [inline source/blog/2018-01-22-CICD-workflow-with-zuul.html.md]: Line 404, 405 could also be put at the beginning of the article.

File: source/blog/2018-01-22-CICD-workflow-with-zuul.html.md [hunk 3]
```diff
@@ -0,0 +1,407 @@
...
+---
+
+The upcoming version of [Zuul](https://docs.openstack.org/infra/zuul/)
+has many new features that allow one to create powerful continuous
+integration and continuous deployment pipelines.
+
+This article presents some mechanisms to create such pipelines using
+a rpm package distribution development workflow as a practical example.
+
+To experiment with the configuration sample,
```
Comment (morucci) [inline source/blog/2018-01-22-CICD-workflow-with-zuul.html.md]: This paragraph could have been placed at the end. Like "experiment Zuul by yourself"

Comment (TristanCacqueray) [reply source/blog/2018-01-22-CICD-workflow-with-zuul.html.md]: Done

File: source/blog/2018-01-22-CICD-workflow-with-zuul.html.md (hunk 1 above)
Comment (TristanCacqueray) [reply source/blog/2018-01-22-CICD-workflow-with-zuul.html.md]: Done

File: source/blog/2018-01-22-CICD-workflow-with-zuul.html.md (hunk 2 above)
Comment (TristanCacqueray) [reply source/blog/2018-01-22-CICD-workflow-with-zuul.html.md]: Done, thanks for the review Fabien!

**Pull request merged**